app/src/main/java/com/fbada/projectone/domain/Car.kt

```diff
@@ -267,8 +272,8 @@ abstract class Alpha(
          * @property price base price of the car
          *
          */
-        val omegaTotalPrice = automaticPetrolPremium() + windscreenPremium()
         override var price: Double = 100.0
+        private val omegaTotalPrice = price + automaticPetrolPremium() + windscreenPremium()
```

Comment:
It's private and is not used within this class. Was that the plan? The `price` just stays 100 always.
Also the doc should be separate per field/property. In your case you've specified `omegaTotalPrice`  docs as part of the `price` docs.

Reply:
Not sure what original thinking was but doesn't make sense... made. not private and rectified doc.